pkg/controllers/deprovisioning/controller.go

```diff
@@ -271,15 +271,15 @@ func (c *Controller) launchReplacementNodes(ctx context.Context, action Command)
 		return fmt.Errorf("cordoning nodes, %w", err)
 	}
 
-	nodeNames, err := c.provisioner.LaunchNodes(ctx, provisioning.LaunchOptions{RecordPodNomination: false}, action.replacementNodes...)
+	nodeNames, err := c.provisioner.LaunchMachines(ctx, provisioning.LaunchOptions{RecordPodNomination: false}, action.replacementMachines...)
```

Comment:
Optional: this api would be cleaner as
```
func LaunchMachines(context.Context, []*Machine, ...provisioning.LaunchOptions)
```
You could simply remove the falsy option here and let it default.